### ISSUE TITLE
Annual throttle of 1,000,000 requests

### ISSUE THREAD

**Issue by sam-a-levy** [vt-arc/wosr#19]
I am trying to use `pull_wos()` using my institutional API credentials (i.e. creating an sid of `auth(username = NULL, password = NULL)` )

This creates an sid no problem and returns responses for `query_wos()` but when I try to use `pull_wos()` I get the following error message

Error: Request denied by Throttle server. Request denied for quantity of 100 for activity "Records" of service "WOK-WS" for id "09039ee8dc22cd24c526909663ccb3a3.27993" on Thursday, March 11, 2021 at 5:53:47 PM UTC.  The following reasons apply:   Reason (1) - amount requested exceeds limit of 1,000,000 per period for throttle REC-AmtPerYear. Details of this throttle are the following: this is an AmountPerPeriod throttle; the period length is 1 year(s); the period restarts after Friday, December 31, 2021 at 11:59:59 PM UTC.

I cannot find any documentation regarding an annual throttle on any of the WoS documentation, so was wondering if you had any insights

**Comment by crew102** [comment]
Yeah, some subscriptions have limits on the number of records they can
download per year. Contact Clarivate analytics and/or your institution to
figure out next steps.
